.github/workflows/doc-index-updater-release.yaml

```diff
@@ -82,16 +82,22 @@ jobs:
         with:
           kustomize: 3.4.0
           command: |
-            set -eux
+            set -ex
```

Comment:
in all these examples, I think we should keep `set -u` in here if we can to protect against unset variables (just changing it to `+u` and back to `-u` around the smallest bits possible)

Reply:
Ah agreed! I'd mixed up +u and -u for this line.